web/src/games/cardtable/CardTable.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FunctionComponent } from 'react';
+import CardContainer from './CardContainer';
+import { IG } from './game';
+import { Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { lightGreen, pink, yellow, purple } from '@material-ui/core/colors';
+
+interface ICardTableProps {
+  G: IG;
+}
+
+const useStyles = makeStyles(() => ({
+  root: {
+    width: '400px',
+    height: '250px',
+    fontSize: '10px',
+    display: 'flex',
+    flexWrap: 'nowrap',
+    justifyContent: 'center',
+    backgroundColor: pink['A100'],
+  },
+  crib: {
+    minWidth: '400px',
+    height: '80px',
+    fontSize: '10px',
+    display: 'flex',
+    flexWrap: 'nowrap',
+    justifyContent: 'center',
+    backgroundColor: lightGreen[900],
+  },
+  playedDown: {
+    //flexGrow: 1,
+    minWidth: '400px',
+    height: '80px',
+    fontSize: '10px',
+    display: 'flex',
+    flexWrap: 'nowrap',
+    justifyContent: 'center',
+    alignContent: 'center',
+    backgroundColor: yellow['A400'],
+  },
+
+  playedTop: {
+    //flexGrow: 1,
+    minWidth: '400px',
+    flexDirection: 'row',
+    height: '80px',
+    fontSize: '10px',
+    display: 'flex',
+    flexWrap: 'nowrap',
+    justifyContent: 'center',
+    alignContent: 'center',
+    backgroundColor: purple['100'],
+  },
+}));
+
+const CardTable: FunctionComponent<ICardTableProps> = (props: ICardTableProps) => {
+  const classes = useStyles();
```

Comment:
Please move all these classes to its own "CardTable.css" file. Then, import it like so:
``import classes from './CardTable.css'``

This will remove the CSS bloat of this file